### DISCUSSION TITLE
image preview not dismiss when zellij tab changed?

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#1216] [1. Q&A] [answered]
for example  I have two tabs in zellij in one tab, I preview a image, then I change to another tab. The image preview not dismiss. Any way to solve it? 

Yazi
    Version: 0.2.5 (c65a14c 2024-04-23)
    OS: linux-x86_64 (unix)
    Debug: false

Emulator
    Emulator.via_env: ("xterm-kitty", "")
    Emulator.via_csi: Ok(Unknown([]))
    Emulator.detect: Kitty

Adaptor
    Adaptor.matches: X11

Desktop
    XDG_SESSION_TYPE: Some("x11")
    WAYLAND_DISPLAY: None
    DISPLAY: Some(":0")

SSH
    shared.in_ssh_connection: false

WSL
    /proc/sys/fs/binfmt_misc/WSLInterop: false

Variables
    SHELL: Some("/usr/bin/fish")
    EDITOR: Some("nvim")
    ZELLIJ_SESSION_NAME: Some("拼搏与梦想")
    YAZI_FILE_ONE: None
    YAZI_CONFIG_HOME: None

file(1)
    Version: Ok(Output { status: ExitStatus(unix_wait_status(0)), stdout: "file-5.45\nmagic file from /usr/share/file/misc/magic\nseccomp support included\n", stderr: "" })

Text Opener
    default: Some(Opener { run: "$EDITOR \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })
    block: Some(Opener { run: "$EDITOR \"$@\"", block: true, orphan: false, desc: "$EDITOR", for_: None, spread: true })

tmux
    TMUX: false

Ueberzug++
    Version: Ok(Output { status: ExitStatus(unix_wait_status(0)), stdout: "ueberzugpp 2.9.6\n", stderr: "" })

--------------------------------------------------
When reporting a bug, please also upload the `yazi.log` log file - only upload the most recent content by time.
You can find it in the "/home/lizhe/.local/state/yazi" directory.

**Comment by sxyazi** [comment, marked as answer]
This is how ueberzug works; it creates a new window that overlays the terminal window, so the terminal's content is covered because they are two separate windows.

It seems there's nothing Yazi can do here. If you want to avoid images covering content, use a terminal that natively supports any image protocol, such as kitty, sixel, or iTerm2.
